code/modules/martial_arts/martial.dm

```diff
@@ -428,5 +446,31 @@
 		return ..()
 	return 0
 
+/obj/screen/combo
+	icon_state = ""
```

Comment:
surely this is null already? why include this

Reply:
makes it obvious that its invisible to begin with. sometimes you write code not because it is needed, but because it makes it easier to understand